python/pyspark/ml/base.py

```diff
@@ -116,3 +116,17 @@ class Model(Transformer):
     """
 
     __metaclass__ = ABCMeta
+
+
+class HasNumFeaturesModel:
+    """
+    Provides getter of the number of features especially for model class
+    It should be mixin with JavaModel.
+    """
+    @property
+    @since("1.7.0")
```

Comment:
Remove since tag.  If this is added to another class in a later version, the since tag will be incorrect.